pkg/apis/vault/v1alpha1/zz_generated.defaults.go

```diff
@@ -14,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !ignore_autogenerated
+
```

Comment:
was this autogenerated change or manual edit? afair the defaulter CLI will override this

Reply:
Manual edit, tried running generate jobs but thankfully the tags were left intact.